xds/src/main/java/io/grpc/xds/internal/sds/SdsProtocolNegotiators.java

```diff
@@ -77,7 +79,14 @@ public static ProtocolNegotiatorFactory clientProtocolNegotiatorFactory(
   public static ProtocolNegotiator serverProtocolNegotiator(
       @Nullable DownstreamTlsContext downstreamTlsContext, int port,
       SynchronizationContext syncContext) {
-    return new ServerSdsProtocolNegotiator(downstreamTlsContext, port, syncContext);
+    XdsClientWrapperForServerSds xdsClientWrapperForServerSds =
+        ServerSdsProtocolNegotiator.getXdsClientWrapperForServerSds(port, syncContext);
+    if (xdsClientWrapperForServerSds == null && downstreamTlsContext == null) {
+      logger.log(Level.INFO, "Fallback to plaintext for server at port {0}", port);
```

Comment:
the port is not used if the xdsClientWrapperForServerSds is null, right?

Reply:
But the server is bound to a port e.g. 8080. As a log message it will be useful to point out to the user that server bound to port 8080 fell back to plaintext for whatever reason. If an application creates, say 5 servers out of which 3 fell back to plaintext because of xDS issues but 2 kept TLS/mTLS because they didn't use xDS it will be useful to log the port in the log.

Reply:
this message is very confusing. there are multiple servers, what is the server here?

Reply:
So not saying it is a "grpc xds-based" server is confusing? 

Given
```
Server serverA = XdsServerBuilder.forPort(80).build();
Server serverB = XdsServerBuilder.forPort(81).build();
Server serverC = NettyServerBuilder.forPort(82).build();
```
The port uniquely identifies the server. So port number 80 or 81 will be mentioned in the log message to point to serverA or serverB resp. I don't understand the confusion.

Reply:
Can you suggest what the message should be?